Podfile.lock

```diff
@@ -500,9 +504,9 @@ SPEC CHECKSUMS:
   WPMediaPicker: 1897f312c7b41114ffd239fb782431ae602134a1
   wpxmlrpc: 6ba55c773cfa27083ae4a2173e69b19f46da98e2
   yoga: 4e71c9a33abf45ba55af55ae9cbc86f4234bb2a9
-  ZendeskSDK: 07977c9aa708ea64461a1b18b7b3cfec3f496785
+  ZendeskSDK: 787414f9240ee6ef8cfe4ea0f00e8b4d01d2d264
```

Comment:
I think `Podfile.lock` needs to be updated. When I installed pods in this branch I got a change on `ZendeskSDK`.

Reply:
Just FYI – this one is ZenDesk's fault :(